app/templates/cases/case/subcases/subcase/overview.hbs

```diff
@@ -37,6 +37,7 @@
         @onSave={{this.saveMandateeData}}
       />
       <Utils::GovernmentAreas::GovernmentAreasPanel
+        @datetimeForGovernmentAreas={{or this.agenda.created this.meeting.plannedStart @model.subcase.created}}
```

Comment:
Geplande start van de meeting (indien er een is) lijkt mij correcter, en dus belangrijker, dan `agenda.created`. Dat laatste is een puur technische datum wanneer de gebruiker de agenda heeft aangemaakt in het systeem.